src/utils/inputs.ts

```diff
@@ -6,8 +6,32 @@ export const checkSum = core.getInput("checksum");
 export const enableCache = core.getInput("enable-cache") === "true";
 export const cacheSuffix = core.getInput("cache-suffix") || "";
 export const cacheLocalPath = getCacheLocalPath();
-export const githubToken = core.getInput("github-token");
 export const cacheDependencyGlob = core.getInput("cache-dependency-glob");
+export const toolBinDir = getToolBinDir();
+export const toolDir = getToolDir();
+export const githubToken = core.getInput("github-token");
+
+function getToolBinDir(): string | undefined {
+  const toolBinDirInput = core.getInput("tool-bin-dir");
+  if (toolBinDirInput !== "") {
+    return toolBinDirInput;
+  }
+  if (process.platform === "win32") {
+    return "D:\\a\\_temp\\uv-tool-bin-dir";
```

Comment:
The docs said it's on `D:` on GitHub-hosted runners. But isn't this setting it on _all_ runners? Is that ok?

Reply:
Good catch. I should rather implement it like the cache, look for `RUNNER_TEMP` first and only use a static value as fallback.

On second thought maybe even fail if `RUNNER_TEMP` is not set. The static fallback path (and drive) might not even exist and then fail with hard to understand error messages

Reply:
Changed it to throw if RUNNER_TEMP is not set.